localstack/services/sqs/models.py

```diff
@@ -496,6 +488,74 @@ def validate_queue_attributes(self, attributes):
             if k not in valid:
                 raise InvalidAttributeName(f"Unknown Attribute {k}.")
 
+    def add_permission(self, actions: list[str], account_ids: list[str], label: str) -> None:
```

Comment:
super minor nit: i think having label as the first argument would be a more consistent API